stock_management/__init__.py

```diff
@@ -1,31 +1,59 @@
 from abc import ABC, abstractmethod
 from enum import Enum
+import time
+import PySimpleGUI as sg
+
 
 class Status(Enum):
-    INITIALIZING = 1,
-    RUNNING = 2,
+    INITIALIZED = 1
+    RUNNING = 2
     FINISHED = 3
 
-class SessionType(Enum):
-    MAIN = 1,
-    TEST = 2
 
 class Session:
-    @abstractmethod
-    def __init__(self, test_events, test_args, session_type):
-        pass
+    def __init__(self, layout_fun, win_name):
```

Comment:
We should aim to have a common `__init__` for the Session class. This is a possibile proposal that might need to be extended to include all possible sessions type.